src/Core/Managed/Shared/Utils.cs

```diff
@@ -74,6 +74,20 @@ public static TimeSpan ValidateDuration(string value)
             return interval;
         }
 
+        /// <summary>
+        /// Returns min DateTimeOffset value if not a valid DateTimeOffset.
+        /// </summary>
+        public static DateTimeOffset ValidateDateTimeOffset(string value)
+        {
+            DateTimeOffset timestamp;
+            if (!DateTimeOffset.TryParse(value, CultureInfo.InvariantCulture, DateTimeStyles.AssumeLocal, out timestamp))
```

Comment:
Would you agree it should be AssumeUniversal?

---
In reply to: [72832994](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/266#discussion_r72832994) [](ancestors = 72832994)